conf/conf.yaml

```diff
@@ -18,6 +18,9 @@ discovery:                       # service discovery center
     host:                        # it's possible to define multiple nacos hosts addresses of the same nacos cluster.
       - "http://127.0.0.1:8848"
     prefix: /nacos/v1/
+    namespace: public            # default namespace for service discovery
```

Comment:
The Nacos discovery in APISIX doesn't configure the namespace in the global scope. Instead, each service can have its individual namespace configuration (the namespace_id)

Reply:
The new code is compatible with the existing APISIX configuration.